EqualityInformationApi/V1/Gateways/EqualityInformationGateway.cs

```diff
@@ -47,5 +46,17 @@ public async Task<EqualityInformation> Update(PatchEqualityInformationObject req
 
             return entity.ToDomain();
         }
+
+        [LogCall]
+        public async Task<EqualityInformation> Get(Guid targetId)
+        {
+            var entity = await _dynamoDbContext.QueryAsync<EqualityInformationDb>(targetId).GetNextSetAsync()
+                .ConfigureAwait(false);
+
+            if (entity == null || !entity.Any())
+                return null;
```

Comment:
shouldn't this throw an exception?

Reply:
If not found, then it should be not found - it's treated in a controller level. Would you rather the controller catches an exception and returns not found?

Reply:
yep :-)